packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -12,14 +12,11 @@ test.describe('Firefox Desktop Sync v3 sign up', () => {
   test.beforeEach(async ({ pages: { login } }) => {
     test.slow();
     email = login.createEmail('sync{id}');
-    await login.clearCache();
```

Comment:
Did this have an effect? It seems like it makes sense that'd you want to emulate a clean cache for sign up tests.

Reply:
It doesnt anymore since using `newPagesForSync` creates a new browser profile.